### PR TITLE
Add spacing around questions

### PR DESCRIPTION
**Pull request by jish**
The questions run into each other and are difficult to separate visually.

Also, the vote button is floating in the middle of the question.

### Before

Screen Shot 2019-05-08 at 2 50 22 PM

### After

Screen Shot 2019-05-08 at 2 55 23 PM

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -159,41 +161,41 @@ h1.header a{
   white-space: pre-line;
 }
 
-.topic .question{
-  margin-top: 20px;
+.topic .question {
+  margin-top: 48px;
```
Comment (jish) [inline public/css/main.css:165]: This line is the change we are going for. Add some spacing in between the questions.

**Comment by jish** [comment]
bump. I'm having a hard time determining when one question ends and the next one begins. Are there some changes you would like to see?

**Comment by jish** [comment]
Bump. What do you think of the proposed changes?

**Comment by johnhuangclever** [comment]
> What we have has emerged,
> A Clever engineer with an urge,
> An old PR on the verge,
> Eh, what the heck, yolo merge

**Pull request closed without merging**